fastlane/Fastfile

```diff
@@ -12,7 +12,7 @@ platform :ios do
     UI.message("Simulator testing device set to '#{device}'")
 
     scan(
-      :workspace => 'HubFramework.xcworkspace',
+      :project => 'HubFramework.xcodeproj',
       :scheme => 'HubFramework',
```

Comment:
Unfortunately not. Xcode doesn’t support running tests with just a target name. You need a scheme 😢